slider/replay.py

```diff
@@ -38,8 +40,9 @@ class Action:
     mouse2 : bool
         is the second mouse button pressed?
     """
-    def __init__(self, offset, position, key1, key2, mouse1, mouse2):
-        self.offset = offset
+    def __init__(self, rel_offset, abs_offset, position, key1, key2, mouse1, mouse2):
+        self.rel_offset = rel_offset
```

Comment:
I was hoping abs and rel were obvious enough abbreviations to reduce the verbosity a little (abs is a built in function after all) but this is a moot point anyway since I think we're in agreement to remove relative and rename absolute back to offset